deeplinkdispatch-processor/src/test/java/com/airbnb/deeplinkdispatch/DocumentorTest.java

```
@@ -0,0 +1,100 @@
+        Documentor documentor = new Documentor(processingEnv);
+        documentor.write(getElements());
+        try {
+            String actual = Files.toString(new File(FILE_PATH), Charsets.UTF_8);
+            String expected = "airbnb://example.com/{foo}/bar\\n|#|\\nSample doc\\n|#|\\nDocClass\\n|##|\\n" +
+                    "airbnb://example.com/{foo}/bar\\n|#|\\n\\n|#|\\nDocClass#DocMethod\\n|##|\\n";
+            assertEquals(expected, actual);
+        } catch (IOException e) {
+            fail();
+        }
+    }
+
+    private List<DeepLinkAnnotatedElement> getElements() {
+        TypeElement element1 = mock(TypeElement.class);
+        Name name1 = mock(Name.class);
+        when(elements.getDocComment(element1)).thenReturn("Sample doc \n @param empty \n @return nothing");
+        when(element1.getSimpleName()).thenReturn(name1);
+        when(name1.toString()).thenReturn("DocClass");
+
+        TypeElement element2 = mock(TypeElement.class);
+        TypeElement element2Enclosed = mock(TypeElement.class);
+        Name name2 = mock(Name.class);
+        Name name2Enclosed = mock(Name.class);
+        when(element2.getSimpleName()).thenReturn(name2);
+        when(name2.toString()).thenReturn("DocMethod");
+        when(element2.getEnclosingElement()).thenReturn(element2Enclosed);
+        when(element2Enclosed.getSimpleName()).thenReturn(name2Enclosed);
+        when(name2Enclosed.toString()).thenReturn("DocClass");
+
+        try {
+            DeepLinkAnnotatedElement deepLinkElement1 =
+                new DeepLinkAnnotatedElement("airbnb://example.com/{foo}/bar", element1, DeepLinkEntry.Type.CLASS);
+            DeepLinkAnnotatedElement deepLinkElement2 =
+                    new DeepLinkAnnotatedElement("airbnb://example.com/{foo}/bar", element2, DeepLinkEntry.Type.METHOD);
+
+            return ImmutableList.of(deepLinkElement1, deepLinkElement2);
+        } catch (MalformedURLException e) {
```

just add `MalformedURLException` to the method `throws` so you don't need this empty catch block